log4j-core/src/main/java/org/apache/logging/log4j/core/appender/mom/jeromq/JeroMqAppender.java

```diff
@@ -225,17 +225,17 @@ public boolean stop(final long timeout, final TimeUnit timeUnit) {
     }
 
     // not public, handy for testing
-    int getSendRcFalse() {
+    synchronized int getSendRcFalse() {
```

Comment:
FWIW IMO, the comments should be Javadoc comments to make them show up in tooling like IDEs.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/3033/commits/9c3d88486c3299bdf2747c928c2f1cd3cf6740fe